src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,175 @@
+    ['trendingCollections'],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+    }
+  )
+
+  useEffect(() => {
+    /* Rotate through Top 5 Collections on 15 second interval */
+    let stale = false
+    if (hovered || stale) return
+    const interval = setInterval(async () => {
+      if (collections) {
+        const nextCollectionIndex = (current + 1) % collections.length
+        setCurrent(nextCollectionIndex)
+      }
+    }, 15_000)
+    return () => {
+      stale = true
+      clearInterval(interval)
+    }
+  }, [current, collections, hovered])
+
+  return (
+    <Box onMouseEnter={() => setHover(true)} onMouseLeave={() => setHover(false)} cursor="pointer" width="full">
+      {collections && collections[current] ? (
+        <Link to={`/nfts/collection/${collections[current].address}`} style={{ textDecoration: 'none' }}>
+          <Box style={{ height: '386px' }}>
+            <div
+              className={styles.bannerWrap}
+              style={{ backgroundImage: `url(${collections[current].bannerImageUrl})` }}
+            >
+              <Box className={styles.bannerOverlay} width="full" />
+              <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
+              <CarouselProgress length={collections.length} currentIndex={current} setCurrent={setCurrent} />
+            </div>
+          </Box>
+        </Link>
+      ) : (
+        <>
+          {/* TODO: Improve Loading State */}
+          <p>Loading</p>
+        </>
+      )}
+    </Box>
+  )
+}
+
+export default Banner
+
+/* Collection Details: displays collection stats within Banner  */
+const CollectionDetails = ({
+  collection,
+  rank,
+  hovered,
+}: {
+  collection: TrendingCollection
+  rank: number
+  hovered: boolean
+}) => (
+  <Box as="section" className={styles.section} paddingTop="40">
+    <Column className={styles.collectionDetails} paddingTop="24">
+      <div className={styles.volumeRank}>#{rank} volume in 24hr</div>
+      <Row>
+        <Box as="span" marginTop="16" className={clsx(header1, styles.collectionName)}>
+          {collection.name}
+        </Box>
+        {collection.isVerified && (
+          <Box as="span" marginTop="24">
+            <VerifiedIcon height="32" width="32" />
+          </Box>
+        )}
+      </Row>
+      <Row className={bodySmall} marginTop="12" color="explicitWhite">
+        <Box>
+          <Box as="span" color="darkGray" marginRight="4">
+            Floor:
+          </Box>
+          {collection.floor ? formatEthPrice(collection.floor.toString()) : '--'} ETH
+        </Box>
+        <Box>
+          {collection.floorChange ? (
+            <Box as="span" color={collection.floorChange > 0 ? 'green200' : 'error'} marginLeft="4">
+              {collection.floorChange > 0 && '+'}
```

Comment:
should there be a `-` here when it's less than 0?

Reply:
API returns a negative valu
<img width="172" alt="negative_floor" src="https://user-images.githubusercontent.com/224071/186273266-b221f988-be3b-4956-a0d2-c17b141d814e.png">
e